x-pack/plugins/cases/public/components/case_view/case_view_tabs.tsx

```diff
@@ -58,10 +88,12 @@ export const CaseViewTabs = React.memo<CaseViewTabsProps>(({ caseData, activeTab
         : []),
       {
         id: CASE_VIEW_PAGE_TABS.FILES,
-        name: FILES_TAB,
+        name: (
```

Comment:
Maybe we can use the `append` property of the tab like:

```
    id: CASE_VIEW_PAGE_TABS.FILES,
    name: FILES_TAB,
    append: {!isLoading && fileStatsData && (
      <StyledNotificationBadge
        data-test-subj="case-view-files-stats-badge"
        color={activeTab === CASE_VIEW_PAGE_TABS.FILES ? 'accent' : 'subdued'}
      >
        {fileStatsData.total > 0 ? fileStatsData.total : 0}
      </StyledNotificationBadge>
    )}
```

Reply:
This one just doesn't work for me 😄

Reply:
Ok, then!